CharlieBackend.Business/Services/AccountService.cs

```diff
@@ -72,7 +72,14 @@ public async Task<Result<AccountDto>> GetAccountCredentialsAsync(AuthenticationD
             {
                 authenticationModel.Password = PasswordHelper.HashPassword(authenticationModel.Password, salt);
 
-                var foundAccount = _mapper.Map<AccountDto>(await _unitOfWork.AccountRepository.GetAccountCredentials(authenticationModel));
+                var cred = await _unitOfWork.AccountRepository.GetAccountCredentials(authenticationModel);
+
+                if(cred == null)
+                {
+                    return Result<AccountDto>.GetError(ErrorCode.Unauthorized, "Email or password is incorrect.");
```

Comment:
Error should be NotFound I think, because if credentials is null, there is no accounts that matches

Reply:
credentials can be null if password is wrong and email exists